docs/layers.md

```diff
@@ -398,14 +397,12 @@ Places with `kind` values of `continent`, `country`, with others added starting
 * `id`: osm_id from OpenStreetMap or Natural Earth id
 * `kind`: normalized values between OpenStreetMap and Natural Earth
 * `population`: population integer values from OpenStreetMap or Natural Earth (`pop_max`)
-* `scalerank`: scalerank value from Natural Earth, and invented for OpenStreetMap
```

Comment:
Now that `scalerank` has been removed and we always calculate `min_zoom`, we should remove `scalerank` from **common** as you propose but also move `min_zoom` from common-optional to common section.